.github/actions/kubectl-keep-nslogs/action.yml

```diff
@@ -37,6 +41,6 @@ runs:
       uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02 # v4.6.2
       with:
         name: >-
-          ${{ inputs.namespace }}_logs.${{ github.run_number }}.${{ github.run_attempt }}
+          ${{ inputs.namespace }}_logs.${{ github.run_number }}.${{ github.run_attempt }}${{ inputs.log_name_suffix }}
```

Comment:
technically it's a new feature so worth a minor bump.

also a bit picky but I prefer we have something like `log_name_identifier` used as
`${{ inputs.namespace }}_${log_name_identifier}_logs` to keep it simple